### ISSUE TITLE
what will be the output of GetStabbleBits.py ? How it look like?

### ISSUE THREAD

**Issue by jafeelv** [asajim/strong-software-based-Physical-Unclonable-Function#7]
what will be the output of GetStabbleBits.py ? How it look like? I am getting the location of strong ones as continuous values (eg: 0-99). Is it because of some errors? 
![image](https://user-images.githubusercontent.com/63637416/226249251-b545786a-8152-4c15-843f-27fe90d0a9f4.png)

**Comment by myndcryme** [comment]
That looks fine.  In fact it might be a very stable SRAM.  I have tested some chips in the past that had outputs similar to yours.  My strongbits reference file I posted is actually not great compared to outputs for other chips I have tested.  Maybe it was a poor example (weak chip) to show but it was already on the breadboard.

Is your output above for the LY62256PL-55LLI ?  Are you having any problems with this particular chip?

**Comment by jafeelv** [comment]
@myndcryme Yeah, I'm getting similar results for LY62256PL-55LLI and 71256SA20TPG chips (ie, continuous values for location of strong ones, starting from 0) But different random values for strong ones.


![Screenshot 2023-03-21 105629](https://user-images.githubusercontent.com/63637416/226525567-6a9fadd5-dc67-4781-92ec-3a120087d480.png)
